wasm/tests/Blockchain/Hedera.test.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright © 2017-2022 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+import "mocha";
+import { assert } from "chai";
+import { Buffer } from "buffer";
+import { TW } from "../../dist";
+import Long = require("long");
+
+describe("Hedera", () => {
+  it("test sign simple transfer Hedera", () => {
```

Comment:
add one address test? not sure if `Address.isValid` will throw exception in ts

Reply:
yes you are right it throws, will probably need an alternative approach, but not sure how we will create a pubkey without throwing in this case